package.json

```diff
@@ -9,14 +9,14 @@
     "Andrew Monks <a@monks.co>"
   ],
   "dependencies": {
-    "@blueoak/list": "^9.0.0",
-    "@npmcli/arborist": "^6.5.0",
+    "@blueoak/list": "^15.0.0",
+    "@npmcli/arborist": "^7.5.4",
```

Comment:
i believe this may be a breaking change due to dropped node versions

Reply:
yup 
https://github.com/npm/cli/blob/latest/workspaces/arborist/package.json#L91-L93
`"^16.14.0 || >=18.0.0"`

Reply:
Given that this version bump doesnt fix https://github.com/jslicense/licensee.js/pull/82 , I am happy to undo it if licensee.js wants to new releases to continue to support node 14

Reply:
It would be a terrible idea to let a dep update force an otherwise unnecessary breaking change imo